kernel/src/process/posix_thread/builder.rs

```diff
@@ -104,6 +106,11 @@ impl PosixThreadBuilder {
         self
     }
 
+    pub(in crate::process) fn set_init_process(mut self) -> Self {
```

Comment:
All builder methods should be named as "properties", not as "actions". So this method should NOT be named `set_init_process`, but `is_init_process`.